backend/plugins/frontline_api/src/modules/integrations/call/utils.ts

```
@@ -0,0 +1,957 @@
+  [-69]:
+    'There is currently a conference going on. Changes cannot be applied at this time',
+  [-70]: 'Login forbidden',
+  [-71]: "The username doesn't exist",
+  [-90]: 'The conference is busy, cannot be edited or deleted',
+  [-98]: 'There are currently digital calls. Failed to apply configuration',
+};
+
+export const toCamelCase = (obj) => {
+  const camelCaseObj = {};
+  for (const key in obj) {
+    if (obj.hasOwnProperty(key)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)